packages/did/src/DidBatcher/FullDidUpdateBuilder.ts

```diff
@@ -124,7 +127,7 @@ export class FullDidUpdateBuilder extends FullDidBuilder {
   public setAuthenticationKey(key: NewDidVerificationKey): this {
     if (this.consumed) {
       throw SDKErrors.ERROR_DID_BUILDER_ERROR(
-        'DID builder has already been consumed.'
+        'DID builder has already been used.'
       )
     }
```

Comment:
-> use `checkConsumed` here and below

Reply:
https://github.com/KILTprotocol/sdk-js/pull/486/commits/c0c5471826feeba8abc6ecb1017c846070b06ce7